<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="analyzertab">
     <attribute name="title">
      <string>Analyzer</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;File1:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>67</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;File2:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>50</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>831</width>
        <height>431</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="spec">
       <attribute name="title">
        <string>Spectral Information</string>
       </attribute>
       <widget class="QPushButton" name="rms">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>95</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>rms</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mfcc">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>95</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>mfcc</string>
        </property>
       </widget>
       <widget class="QPushButton" name="chroma">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>95</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>chroma</string>
        </property>
       </widget>
       <widget class="QPushButton" name="centroid">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>95</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>centroid</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_a">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>331</width>
          <height>221</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_b">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>110</y>
          <width>351</width>
          <height>221</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_a1">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>340</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Spectral Information of File1.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_b1">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>340</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Spectral Information of File2.</string>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>80</y>
          <width>20</width>
          <height>261</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="fna">
       <attribute name="title">
        <string>Frequency and amplitude</string>
       </attribute>
       <widget class="QPushButton" name="button1">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>50</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>View graph</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button2">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>50</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>View graph</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Frequency and amplitude of file1:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>50</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Frequency and amplitude of file2:</string>
        </property>
       </widget>
       <widget class="QLabel" name="fnalabel1">
        <property name="geometry">
         <rect>
          <x>26</x>
          <y>110</y>
          <width>331</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="fnalabel2">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>110</y>
          <width>331</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>100</y>
          <width>20</width>
          <height>261</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="bpm">
       <attribute name="title">
        <string>Beats per minute</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>70</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>File2:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>67</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>File1:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Estimated Bpm:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>340</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Estimated Bpm:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bpm1">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>95</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>View Graph</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bpm2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>70</y>
          <width>95</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>View Graph</string>
        </property>
       </widget>
       <widget class="QLabel" name="bpmlabel1">
        <property name="geometry">
         <rect>
          <x>46</x>
          <y>120</y>
          <width>281</width>
          <height>201</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="bpmlabel2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>120</y>
          <width>281</width>
          <height>201</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>100</y>
          <width>20</width>
          <height>231</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="bpml1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>340</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="bpml2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>340</y>
          <width>231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>AnalyseAndCompare</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mixertab">
     <attribute name="title">
      <string>Mixer</string>
     </attribute>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>20</width>
        <height>451</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cut:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cut:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>190</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>To:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>190</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>To:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Crossfade:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Crossfade:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Begin:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>End:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Begin:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>270</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>End:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="apply1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apply3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="undo1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="undo2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>390</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>490</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge and Export</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apply2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apply4">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>390</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Sample</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>390</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Sample</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
